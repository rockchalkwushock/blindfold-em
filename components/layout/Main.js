const Main = ({ children }) => (
  <main>
    {children}
    <style jsx>{`
      main {
        background-color: lightblue;
        display: flex;
        flex-direction: column;
        height: 65vh;
        padding: 2em 3em;
      }
    `}</style>
  </main>
)

export default Main
