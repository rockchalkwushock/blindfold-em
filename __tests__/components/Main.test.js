import React from 'react'
import { mount } from 'enzyme'

import Main from '../../components/layout/Main'

const Test = () => (
  <Main>
    <div>1</div>
    <div>2</div>
    <div>3</div>
  </Main>
)

describe('Component: <Main />', () => {
  test('should mount without exploding', () => {
    const tree = mount(<Test />)
    expect(tree).toMatchSnapshot()
  })
})
