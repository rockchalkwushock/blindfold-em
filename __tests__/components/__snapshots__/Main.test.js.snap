// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Main /> should mount without exploding 1`] = `
<Test>
  <Main>
    <main
      className="jsx-3083400802"
    >
      <div>
        1
      </div>
      <div>
        2
      </div>
      <div>
        3
      </div>
      <JSXStyle
        css="main.jsx-3083400802{background-color:lightblue;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;height:65vh;padding:2em 3em;}"
        styleId="3083400802"
      />
    </main>
  </Main>
</Test>
`;
